client.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
We discussed it verbally, but I formalize here a request for writing some test for this functionality.